modules/unsupported/cql-json/pom.xml

```
@@ -22,6 +22,12 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.geotools</groupId>
+      <artifactId>gt-geojson</artifactId>
+      <version>27-SNAPSHOT</version>
```

Don't use a fixed version, it will break after the next major release
```suggestion
      <version>${project.version}</version>
```

---

Thanks, unforced error when using IDE to populate dependencies.  FIXED